deploy/Dockerfile

```diff
@@ -1,11 +1,19 @@
 FROM alpine:3.10
 MAINTAINER dengnan@google.com vmarmol@google.com vishnuk@google.com jimmidyson@gmail.com stclair@google.com
 
-RUN apk --no-cache add libc6-compat device-mapper findutils zfs && \
-    apk --no-cache add thin-provisioning-tools --repository http://dl-3.alpinelinux.org/alpine/edge/main/ && \
+RUN apk --no-cache add libc6-compat=1.1.22-r3 device-mapper==2.02.184-r0 findutils=4.6.0-r1 zfs=0.8.2-r0  build-base=0.5-r1 linux-headers=4.19.36-r0 && \
```

Comment:
why are we pinning dependencies to specific versions?  Is this required by the installation of libpfm4?

Reply:
libpfm4 does not require any specific version of packages. I added versions to control installed versions. Should I go back to previous state here?

Reply:
yeah, go back to the previous state.  Otherwise, i'll forget, and we will have packages with security problems...